nursery_store.go

```diff
@@ -394,7 +394,7 @@ func (ns *nurseryStore) CribToKinder(bby *babyOutput) error {
 
 		// Retrieve or create a height-channel bucket corresponding to
 		// the kidOutput's maturity height.
-		hghtChanBucketCsv, err := ns.createHeightChanBucket(tx,
+		heightChanBucketCsv, err := ns.createHeightChanBucket(tx,
```

Comment:
This (and all the other instances below it), aren't really spelling mistakes. They're purposeful variable names.